### PR TITLE
Minor savedata dialog changes

### PR DESCRIPTION
**Pull request by Megamouse**
- Replace title with subtitle. The title is always the game name and is obsolete since the game should be running anyway.
- Show the date of modification and the size of the savegame where the subtitle was shown before
- Keep the focus on the last selected element if you declined the confirmation dialog

Old:

image

New:

image
image

Reference:

image

### CODE REVIEW

**Comment by Megamouse** [comment]
Adjusted some wording and changed datetime to a more localized version (albeit less PS3-esque)

**Comment by Megamouse** [comment]
Added size info. It may not be important but why not.

File: rpcs3/Emu/Cell/Modules/cellSaveData.cpp [hunk 1]
```diff
@@ -53,6 +55,23 @@ SaveDialogBase::~SaveDialogBase()
...
 
+std::string SaveDataEntry::date() const
+{
+	return date_time::fmt_time("%c", mtime);
+}
+
+std::string SaveDataEntry::data_size() const
+{
+	std::string metric = "KB";
+	u64 sz = static_cast<u64>(std::ceil(size / 1000.0));
```
Comment (elad335) [inline rpcs3/Emu/Cell/Modules/cellSaveData.cpp:66]: utils::aligned_div is what you're looking for.

Comment (Megamouse) [reply rpcs3/Emu/Cell/Modules/cellSaveData.cpp]: done

**Pull request merged**